experimental/dds/tree/src/migration-shim/shimDeltaConnection.ts

```diff
@@ -119,16 +106,4 @@ export class StampDeltaConnection implements IDeltaConnection {
 	public dirty(): void {
 		this.deltaConnection.dirty();
 	}
-
-	/**
-	 * Passes through to the underlying delta connection.
-	 *
-	 * @deprecated There is no replacement for this, its functionality is no longer needed at this layer.
-	 * It will be removed in a future release, sometime after 2.0.0-internal.8.0.0
-	 *
-	 * @privateRemarks This needs to be more thoroughly thought through. What happens when the source handle is changed?
```

Comment:
@tyler-cai-microsoft - Do you remember what exactly your concern was here?  https://github.com/microsoft/FluidFramework/pull/17709/files#diff-965d573e32dba6b55c5312a8ae2a6d8b4457227860fa892644c8cc402e246cb3R57

I don't fully grasp the data migration scenario, when this shim would kick in, etc.  With the new scheme where the ContainerRuntime peeks into each op payload when processing, and tells GC about new references at that point, do you have any more concerns around data migration?

Reply:
I don't think so